### DISCUSSION TITLE
Style Guide

### DISCUSSION THREAD

**Discussion by jeffrey-tafe** [gh44922023/Tafe230221#3] [General]
Do we have a standard font and font sizes for elements within the application?

**Comment by BrodieBrown1998** [comment]
Use the one built in.

**Reply by jeffrey-tafe** [reply]
Do you agree @gh44922023 ?

**Comment by gh44922023** [comment]
Agreed for standard font, colour for program consistency
However Font size may need to be adjusted to fit all controls in output working area so output is displayed correctly.
